chatbot_core/v2/__init__.py

```diff
@@ -445,6 +454,15 @@ def _handle_next_shout(self):
     def _pause_responses(self, duration: int = 5):
         pass
 
+    def stop_shout_thread(self):
+        if self.shout_thread:
+            self.shout_thread.cancel()
+            self.shout_thread = None
+
     def shutdown(self):
         self.shout_thread.cancel()
         self.shout_thread.join()
+
+    def stop(self):
+        self.stop_shout_thread()
+        super().stop()
```

Comment:
Please make this explicit:
```suggestion
        KlatAPIMQ.stop(self)
```

Reply:
> Please make this explicit:

resolved